network_provisioning/tool/esp_prov/prov/__init__.py

```diff
@@ -0,0 +1,8 @@
+# SPDX-FileCopyrightText: 2018-2022 Espressif Systems (Shanghai) CO LTD
```

Comment:
I think we should also add the `requirement.txt` file which list the requirements for the python tool.
I think the requirements are currently present in `requirements.ci.txt`, but then we may have to give appropriate directions to install those
e.g., `bleak` etc.

Reply:
I have already mentioned it in the README file. To use this python tool script, the user should run `bash install.sh --enable-ttfw` or `bash install.sh --enable-pytest` under `$IDF_PATH` directory.